src/features/defi/components/Approve/Approve.tsx

```diff
@@ -84,8 +82,25 @@ export const Approve = ({
   )
 
   const approveHeaderTranslation: TextPropTypes['translation'] = useMemo(
-    () => ['modals.approve.header', { asset: asset.name, spenderName }],
-    [asset.name, spenderName],
+    () => [
+      isReset ? 'modals.approve.resetHeader' : 'modals.approve.header',
+      { asset: asset.name, spenderName },
+    ],
+    [asset.name, isReset, spenderName],
+  )
+
+  const approveCopy = useMemo(() => {
+    if (isReset) return 'common.reset'
```

Comment:
Not a mistake:

- We would never see the "approved" step here and instantly go to the next step (approve) - the only flow where "approved" is displayed is in the case of UNI-V2 LP deposit, where we are depositing two tokens, each with their own step, however our only first-class support is ETH/FOX LP, so reset will never be a concern there
- Accidentally, the past tense of reset is... reset